app/controllers/FaciaToolController.scala

```diff
@@ -55,25 +56,6 @@ class FaciaToolController(
     }
   }
 
```

Comment:
To make the endpoint usage across tool versions explicit, we've moved getCollections to the V2 controller.